source/_integrations/miele.markdown

```diff
@@ -0,0 +1,155 @@
+---
+title: Miele
+description: Instructions on how to set up the Miele integration within Home Assistant.
+ha_category:
+  - Hub
+  - Sensor
+ha_iot_class: Cloud Push
+ha_release: '2025.5'
+ha_domain: miele
+ha_codeowners:
+  - '@astrandb'
+ha_config_flow: true
+ha_platforms:
+  - sensor
+ha_integration_type: integration
+---
+
+The Miele integration allows users to integrate their home appliances using the [official 3rd party API](https://www.miele.com/developer).
+
+Miele is known as a manufacturer of premium appliances for cooking, laundry care and floorcare.
```

Comment:
```suggestion
Miele is known as a manufacturer of premium appliances for cooking, laundry care, and floorcare.
```

Reply:
I always miss the Oxford comma. It is considered bad style in my native language Swedish...

Reply:
aha, sorry to cause you style-pain :smile: I can't think of a specific one right now, but I'm sure I also have my style habits..